tests/test-resource-groups/main.tf

```diff
@@ -0,0 +1,12 @@
+provider "aws" {
+  region = "us-east-1"
+}
```

Comment:
This should be necessary, I don't think? We can let the provider default to the user's config.
```suggestion
```

Reply:
resolved. it's not i was just specifying a provider as a chance to practice/write additional stuff